code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -121,7 +121,11 @@ var/list/nest_locations = list()
 					D.attackby(CB,src)
 					qdel(CB)
 
-/mob/living/simple_animal/hostile/alien/CanAttack(var/atom/the_target)//they don't kill mindless monkeys so they can drag them to nests with a higher chance of a successful impregnation.
+/mob/living/simple_animal/hostile/alien/CanAttack(var/atom/the_target)
+	if(isalien(the_target))
```

Comment:
Carbon aliens don't make use of factions. They are neutral by default and I'm afraid of breaking things by reworking that.

isalien() checks for both simple animals and carbons.